### ISSUE TITLE
plugin_python.dll fails to load

### ISSUE THREAD

**Issue by dystopiat** [ModOrganizer2/modorganizer#1998]
## The problem:
Briefly describe the issue you are experiencing. Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our [Discord server](https://discord.gg/ewUVAqyrQX).

## To Reproduce:

Switch from one instance to another: "Mod Organizer failed to load the plugin 'plugin_python' last time it was started.

I get this message every time.

## Environment:

* Mod Organizer Version that exhibits the issue: 2.5.1rc2
* Last Mod Organizer Version that did not exhibit the issue (if applicable): 2.4.4
* Desktop OS/version used to run Mod Organizer: Windows 10

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Despite the pop-up, the log says the plugin loaded:
https://gist.github.com/dystopiat/dc4b10c8aaa95387a9999853f3c6bb86

### MO Interface:
whoops I put both logs in the link above. They are short.

**Comment by AnyOldName3** [comment]
Can you please turn your log level up to Debug and then do this again to get a more detailed log.

**Comment by dystopiat** [comment]
Yeah, I changed log level on the instance I switched from, my apologies. I've made an edit above:
Edit: It seems the issue is the Bethesda Plugin Manager for Mod Organizer: https://www.nexusmods.com/skyrimspecialedition/mods/111236. When I removed that plugin I no longer get the error message.

Should I re-add the Plugin Manager and provide new logs?
